src/ext/grep.lisp

```diff
@@ -70,12 +70,14 @@
 (defvar *last-query* "git grep -nH ")
 (defvar *last-directory* nil)
 
-(define-command grep (query &optional (directory (buffer-directory)))
-    ((prompt-for-string "" :initial-value *last-query* :history-symbol 'grep)
-     (princ-to-string (prompt-for-directory "Directory: "
-                                            :directory (if *last-directory*
-                                                           (princ-to-string *last-directory*)
-                                                           (buffer-directory)))))
+(define-command grep
+  (query &optional (directory (buffer-directory)))
+  ((prompt-for-string "" :initial-value *last-query* :history-symbol 'grep)
```

Comment:
This would be appreciated if you could code format it.
Indenting with lem makes it 4 widths.